core/templates/components/skill-selector/select-skill-modal.component.spec.ts

```diff
@@ -94,7 +95,8 @@ describe('Select Skill Modal', () => {
   it('should close modal on confirm', () => {
     spyOn(ngbActiveModal, 'close');
     componentInstance.selectedSkillId = '2';
-    let totalSkills = [];
+    let totalSkills:
+    (ShortSkillSummary |SkillSummaryBackendDict | SkillSummary)[] = [];
```

Comment:
```suggestion
    let totalSkills: (
      ShortSkillSummary | SkillSummaryBackendDict | SkillSummary
    )[] = [];
```

Why does this have some many different types?